src/js/components/Card.js

```diff
@@ -85,21 +113,59 @@ export default class Card extends Component {
       CLASS_ROOT,
       {
         [`${CLASS_ROOT}--direction-${direction}`]: direction,
-        [`${CLASS_ROOT}--selectable`]: (onClick || video)
+        [`${CLASS_ROOT}--selectable`]: (onClick || video),
+        [`${CLASS_ROOT}--${textSize}`]: textSize
       },
       className
     );
 
+    let labelTag;
+    let headingTag;
+    let paragraphSize;
+    let descriptionSize;
+    if (textSize === 'xlarge') {
```

Comment:
This if/else block might be a bit cleaner as a const object at the top of the file, keyed by the size.